cms/jinja2/templates/pages/release_calendar/release_calendar_page.html

```diff
@@ -0,0 +1,145 @@
+{% extends "templates/base_page.html" %}
+
+{% from "components/table-of-contents/_macro.njk" import onsTableOfContents %}
+
+{% block main %}
+    <div class="ons-u-fs-m ons-u-mt-s ons-u-pb-xxs release__document-type">Release</div>
+    <h1 class="ons-u-fs-xxxl ons-u-mb-m">
+        <span>{{ page.title }}</span>
+        {% if page.is_accredited %}
+            <a href="https://uksa.statisticsauthority.gov.uk/about-the-authority/uk-statistical-system/types-of-official-statistics/" class="national-statistics__link ons-u-ml-s">
+                <img src="https://cdn.ons.gov.uk/assets/images/ons-logo/kitemark/v2/uksa-kitemark-en.svg" alt="UK Statistics Authority Kitemark" class="osr__logo" width="90">
+            </a>
+        {% endif %}
+    </h1>
+
+    <div class="ons-grid ons-u-ml-no">
+        {% if page.is_census %}
+            <div class="ons-u-pb-s">
+                <img src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-census-2021-purple-landscape.svg" alt="Census 2021 logo" width="171">
+            </div>
+        {% endif %}
+        {% block release_meta %}
+            <div class="ons-grid__col ons-grid__col--sticky@m ons-col-4@m ons-col-12@s ons-u-p-no">
+                <span class="ons-u-fs-r--b">{{ _("Release date") }}:</span>
+                <span class="ons-u-nowrap">{{ page.release_date|date("j F Y g:ia") or page.release_date_text }}</span>
+            </div>
+            {% if page.next_release_date or page.next_release_text %}
+                <div class="ons-grid__col ons-col-8@m ons-col-12@s ons-u-p-no@xxs@m">
+                    <span class="ons-u-fs-r--b">{{ _("Next release") }}:</span>
+                    <span class="ons-u-nowrap">{{ page.next_release_date|date("j F Y g:ia") or page.next_release_text }}</span>
+                </div>
+            {% endif %}
+
+            <p></p>{# added empty p for some margin #}
```

Comment:
Nit: This feels like a hack - is there a better way to add some margin? Perhaps with CSS or a `<br>`?

Reply:
I am not doing all the front-end here. We have FE tasks for heroes